ee/clickhouse/sql/trends/breakdown.py

```diff
@@ -106,10 +106,8 @@
         SELECT toStartOfDay(toDateTime(timestamp), %(timezone)s) as timestamp FROM events e WHERE team_id = %(team_id)s {parsed_date_from_prev_range} {parsed_date_to} GROUP BY timestamp
     ) d
     CROSS JOIN (
-        SELECT toStartOfDay(toDateTime(timestamp), %(timezone)s) as timestamp, pdi.person_id AS person_id, {breakdown_value} as breakdown_value
+        SELECT toStartOfDay(toDateTime(timestamp), %(timezone)s) as timestamp, {person_id_alias}.person_id AS person_id, {breakdown_value} as breakdown_value
         FROM events e
-        INNER JOIN ({GET_TEAM_PERSON_DISTINCT_IDS}) as pdi
```

Comment:
this was a nasty bug. Always having these here meant we'll have two pdi joins when we also want to join persons

https://sentry.io/organizations/posthog2/issues/3154668145/?query=is%3Aunresolved